docs/en/quickstart.md

```diff
@@ -197,13 +207,12 @@ The Ultralytics command line interface (CLI) allows for simple single-line comma
         ```
 
 !!! Warning "Warning"
-
     Arguments must be passed as `arg=val` pairs, split by an equals `=` sign and delimited by spaces between pairs. Do not use `--` argument prefixes or commas `,` between arguments.
 
-    - `yolo predict model=yolov8n.pt imgsz=640 conf=0.25`  ✅
-    - `yolo predict model yolov8n.pt imgsz 640 conf 0.25`  ❌ (missing `=`)
-    - `yolo predict model=yolov8n.pt, imgsz=640, conf=0.25`  ❌ (do not use `,`)
-    - `yolo predict --model yolov8n.pt --imgsz 640 --conf 0.25`  ❌ (do not use `--`)
+    - `yolo predict model=yolov8n.pt imgsz=640 conf=0.25` ✅
```

Comment:
The manually added `&nbsp;` is being removed by `mdformat`, which might be possible to resolve by replacing with `<div>&nbsp;</div>` for mdformat to ignore

Reply:
This is ok, thanks!